mne/io/ant/ant.py

```diff
@@ -93,16 +94,12 @@ def __init__(
         bipolars: list[str] | tuple[str, ...] | None,
         impedance_annotation: str,
         *,
+        encoding: str = "latin-1",
         preload: bool | NDArray,
         verbose=None,
     ) -> None:
         logger.info("Reading ANT file %s", fname)
-        if importlib.util.find_spec("antio") is None:
-            raise ImportError(
-                "Missing optional dependency 'antio'. Use pip or conda to install "
-                "'antio'."
-            )
-        check_version("antio", "0.3.0")
+        _soft_import("antio", "reading ANT files", min_version="0.5.0")
```

Comment:
Realized this `check_version` was a noop -- it returns a `bool`, doesn't raise an error. So I just improved `_soft_import` to have a `min_version` arg which is really the goal here I think.